src/hooks/use-expiration-date.ts

```diff
@@ -38,9 +38,26 @@ export function useExpirationDate( snapshotDate: number ) {
 		} ),
 		{ format, delimiter: ', ' }
 	);
+
+	// Create static translation values
+
+	const countDownTranslated = [
+		{ key: 'days', value: __( 'days' ) },
+		{ key: 'day', value: __( 'day' ) },
+		{ key: 'hours', value: __( 'hours' ) },
+		{ key: 'hour', value: __( 'hour' ) },
+		{ key: 'minutes', value: __( 'minutes' ) },
+		{ key: 'minute', value: __( 'minute' ) },
+	];
+
+	// Map the translated values to the countDown string
+	const translatedCountDown = countDownTranslated.reduce( ( acc, { key, value } ) => {
+		return acc.replace( key, value );
+	}, countDown );
```

Comment:
Nice!, I tested it and it works great.
As a suggestion, here is another alternative that I think is easier to understand:

```diff
- const countDown = formatDuration(
+ let countDown = formatDuration(
```

```suggestion
	const countDownTranslated = {
		days: __( 'days' ),
		day: __( 'day' ),
		hours: __( 'hours' ),
		hour: __( 'hour' ),
		minutes: __( 'minutes' ),
		minute: __( 'minute' ),
	};

	// Replace each translated word to the countDown string
	for ( const [ key, value ] of Object.entries( countDownTranslated ) ) {
		countDown = countDown.replace( key, value );
	}
```